dartagnan/src/test/java/com/dat3m/dartagnan/program/analysis/alias/AliasTest.java

```diff
@@ -0,0 +1,71 @@
+package com.dat3m.dartagnan.program.analysis.alias;
+
+import com.dat3m.dartagnan.configuration.Arch;
+import com.dat3m.dartagnan.parsers.cat.ParserCat;
+import com.dat3m.dartagnan.parsers.program.ProgramParser;
+import com.dat3m.dartagnan.program.Program;
+import com.dat3m.dartagnan.verification.Context;
+import com.dat3m.dartagnan.verification.VerificationTask;
+import com.dat3m.dartagnan.wmm.Relation;
+import com.dat3m.dartagnan.wmm.Wmm;
+import com.dat3m.dartagnan.wmm.analysis.RelationAnalysis;
+import org.junit.Test;
+import org.sosy_lab.common.configuration.Configuration;
+import org.sosy_lab.common.configuration.InvalidConfigurationException;
+
+import java.io.File;
+import java.util.EnumSet;
+import java.util.LinkedList;
+import java.util.List;
+
+import static com.dat3m.dartagnan.configuration.OptionNames.ENABLE_EXTENDED_RELATION_ANALYSIS;
+import static com.dat3m.dartagnan.configuration.Property.PROGRAM_SPEC;
+import static com.dat3m.dartagnan.utils.ResourceHelper.getRootPath;
+import static com.dat3m.dartagnan.utils.ResourceHelper.getTestResourcePath;
+import static com.dat3m.dartagnan.verification.solving.ModelChecker.*;
+import static com.dat3m.dartagnan.wmm.RelationNameRepository.LOC;
+import static org.junit.Assert.assertEquals;
+
+public class AliasTest {
```

Comment:
Since this is an awkward test, can we add a comment about it? Maybe even rename it to `AliasDeterminismTest` or something along those lines.